pkg/winch/auth.go

```diff
@@ -52,12 +48,9 @@ func (f *AuthFactory) Get(configSource *pb.AuthSource) (tokenauth.Source, error)
 	var source tokenauth.Source
 	var err error
 
-	ctx, cancel := context.WithTimeout(context.Background(), *fAuthTimeout)
-	defer cancel()
-
 	switch s := configSource.GetType().(type) {
 	case *pb.AuthSource_Kube:
-		source, err = k8sauth.New(ctx, configSource.Name, s.Kube.Path, s.Kube.User)
+		source, err = k8sauth.New(context.Background(), configSource.Name, s.Kube.Path, s.Kube.User)
```

Comment:
some timeout will be nice here

Reply:
Right, I have reverted tihs revert since we are actually handling contexts properly :).